CHANGELOG.md

```diff
@@ -15,6 +16,86 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the previous public API version
 
+# r2.1
+
+## Release Notes
+
+This **public release** contains the definition and documentation of
+* location-verification v2.0.0-rc.1
+* location-retrieval v0.4.0-rc.1
+* geofencing-subscriptions v0.4.0-rc.1
+
+The API definition(s) are based on
+* Commonalities v0.5.0
+* Identity and Consent Management v0.2.1
+
+**Changelog since r1.2**
+
+
+* Full Changelog with the list of PRs and contributors: https://github.com/camaraproject/DeviceLocation/compare/r1.2...r2.1
+
+## location-verification v2.0.0-rc.1

```

Comment:
It is implicit that there are breaking changes, but a note about the kind of breaking changes compared to the previous version would be good (especially as they are quite limited).